core/kazoo_ast/src/fs_prop_usage.erl

```diff
@@ -0,0 +1,648 @@
+-module(fs_prop_usage).
+
+%% module for parsing ecallmgr modules, looking for Props from FreeSWITCH
+
+-export([process/0, process/1
+        ,to_header_file/0
+        ]).
+
+-include_lib("kazoo_ast/include/kz_ast.hrl").
+-include_lib("kazoo/src/kz_json.hrl").
+
+-define(DEBUG(_Fmt, _Args), 'ok').
+%% -define(DEBUG(Fmt, Args), io:format([$~, $p, $  | Fmt], [?LINE | Args])).
+
+to_header_file() ->
+    Usage = process(),
+    write_usage_to_header(Usage).
+
+write_usage_to_header(Usage) ->
+    {'ok', IO} = file:open(event_filter_filename()
```

Comment:
I wanted control over the formatting of the hrl file, and `~p` doesn't format as desired. Hence the open + write.